### DISCUSSION TITLE
Where are Aave interfaces located

### DISCUSSION THREAD

**Discussion by AlbertoSinigaglia** [smartcontractkit/full-blockchain-solidity-course-py#43] [Q&A] [answered]
Hi, just a quick clarification, where did you find the Aave contract interface? If I check the Aave profile on github I can find only this:
```solidity
// SPDX-License-Identifier: agpl-3.0
pragma solidity 0.6.12;

interface IWETHGateway {
  function depositETH(
    address lendingPool,
    address onBehalfOf,
    uint16 referralCode
  ) external payable;

  function withdrawETH(
    address lendingPool,
    uint256 amount,
    address onBehalfOf
  ) external;

  function repayETH(
    address lendingPool,
    uint256 amount,
    uint256 rateMode,
    address onBehalfOf
  ) external payable;

  function borrowETH(
    address lendingPool,
    uint256 amount,
    uint256 interesRateMode,
    uint16 referralCode
  ) external;
}
```
But it's pretty different to your one

**Comment by ChakibHi** [comment]
For my case, I used the following directories: 
- https://github.com/aave/protocol-v2/tree/master/contracts/misc/interfaces
- https://github.com/aave/protocol-v2/tree/master/contracts/interfaces

**Reply by AlbertoSinigaglia** [reply]
sorry but none of those intarfaces matched this one: [https://kovan.etherscan.io/address/0xd0a1e359811322d97991e03f863a0c30c2cf029c#code](https://kovan.etherscan.io/address/0xd0a1e359811322d97991e03f863a0c30c2cf029c#code)

**Reply by ChakibHi** [reply]
`IWETH.sol` interface match the contract you mentioned (just the `transfer()` missing but under the hood it is the same as `transferFrom()`)

**Comment by PatrickAlphaC** [comment, marked as answer]
You can find it in the [Aave Docs Interface Section](https://docs.aave.com/developers/the-core-protocol/lendingpool/ilendingpool)
